Dockerized Central 0.13.5 is not starting

Hi, the current docker image for the central (0.13.5) is not working, even with the improvements coming from #616 .

After some investigation, I think I'm able to sport the issue:
the very first line in the `docker-entrypoint.sh` contains a ^M at the end, 

`#!/bin/bash^M`

Just remove it and the image works again!

To prove it, run the following command:
`docker run -it -u root --entrypoint '' glowroot/glowroot-central:0.13.5 bash`

A bash is opened inside the image. Try to run the `docker-entrypoint.sh` file:
`/usr/local/bin/docker-entrypoint.sh`

and you get an error:
`bash: docker-entrypoint.sh: /bin/bash^M: bad interpreter: No such file or directory`

Now remove all `^M` by running this `sed` command:
`sed -i -e 's/\r$//' /usr/local/bin/docker-entrypoint.sh`

Try to run again the `docker-entrypoint.sh`  -> it works!!

Would it be possible to regenerate the docker image of the central without the trailing ^M chars ?

Thanks a lot

```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:22 min
[INFO] Finished at: 2020-03-04T13:08:33+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.6:bower (bower-prune) on project glowroot-ui: Failed to run task: 'bower prune' failed. org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :glowroot-ui
```
My apologies, I know nothing about java or maven, I just need the dockerfile working :<

I could also use some help trying to build the image.

https://search.maven.org/search?q=a:grails-taglib

The above maven central URL says 3.2.12 is the latest version of grails-taglib.
But the build is asking for something newer than current?  It sure looks like it.  The build error below is looking for:  org.grails:grails-taglib:jar:3.3.2.

Any suggestions?

`[ERROR] Failed to execute goal on project glowroot-agent-grails-plugin: Could not resolve dependencies for project org.glowroot:glowroot-agent-grails-plugin:jar:0.13.7-SNAPSHOT: The following artifacts could not be resolved: org.grails:grails-taglib:jar:3.3.2, org.grails:grails-web-gsp:jar:3.3.2, org.grails:grails-web-sitemesh:jar:3.3.2, org.grails.plugins:converters:jar:3.3.1, org.grails.plugins:async:jar:3.3.2, org.grails.plugins:gsp:jar:3.3.2: Could not find artifact org.grails:grails-taglib:jar:3.3.2 in epo-maven-dev (https://artifactory.fis.dev/artifactory/epo-maven-dev) -> [Help 1]
`

--- a/.gitattributes
+++ b/.gitattributes
@@ -1,1 +1,2 @@
 * text=auto
+*.sh text eol=lf

